.github/workflows/build.yml

```diff
@@ -163,6 +163,7 @@ jobs:
         with:
           path: ${{ env.ORCHESTRATOR_HOME }}
           key: orchestrator-${{ env.TODAY }}
+          enableCrossOsArchive: true
```

Comment:
Please note that this option may not work for now, see [this ticket](https://sonarsource.atlassian.net/browse/BUILD-9492).